parent/pom.xml

```diff
@@ -467,6 +468,53 @@
             </build>
         </profile>
 
+        <!--
+            Profile that activates automatically when maven.test.skip is set to true and excludes the test phase
```

Comment:
Comment is a bit confusing, let's rephrase:
```
Profile to exclude the test phase from maven lifecycle when maven.test.skip is set. It is required to mitigate the issue with test-scope dependencies in non tested builds
```
No need for additional info about automatic activation (previous profile has no such info too).